modules/acquia_cms_common/composer.json

```diff
@@ -64,7 +64,8 @@
         "drupal/acquia_cms_starter": "<1.3.3",
         "drupal/acquia_cms_toolbar": "<1.4",
         "drupal/acquia_cms_tour": "2.0.* || <1.3.2",
-        "drupal/acquia_cms_video": "<1.4"
+        "drupal/acquia_cms_video": "<1.4",
+        "acquia/acquia_cms": "<1.5.2"
```

Comment:
@rajeshreeputra I think this is not required. As we are already adding conflict [here](https://github.com/acquia/acquia_cms/pull/1371/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R42).

Reply:
We have added in both version because we releases acquia_cms_common next version before 1.5.x then there will be no patch available. To make sure core patch existing after upgrade we have added in both places.